### PR TITLE
docs: chair-group initial commit

### LINKED ISSUE #67: Chair Group Proposal
Proposal to create : 

- Chair group to manage and organize all activities of the working group.

Draft of chair group proposal [here](https://github.com/unicode-org/message-format-wg/pull/66)

### PR DESCRIPTION
**Pull request by romulocintra**
Closes #67 

Draft for chair group "rules" and "definition" [source document](https://docs.google.com/document/d/1U6PiFopoOqPyAgJ_KSzEfZOyKIcCGA-qqb_PyRrv_Mc/edit?ts=5e5dd569)

### CODE REVIEW

File: guidelines/chair-group.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+- Take care of the minutes of the meetings, including recording of all decisions. The minutes and presentations given in WG should be available in the WG Drive
+
+- Participate and on the preparation of technical drafts or documentation.
+
+
+#### Participate
+
+Share your disposal to participate using one of the following using one of the following channels:
+
+- Send an email [email](mailto:message-format-wg@chromium.org)
```
Comment (sffc) [inline guidelines/chair-group.md]: I can make a new alias for the chair group.  Probably good, to keep traffic down on the main list.

File: guidelines/chair-group.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+- Use the possibility of e-mail or  synchronization meetings to discuss issues or organize WG work.
+
+- Create and update events and agenda.
+
+- Participate in activities to share and promote the WG and all possible synergies that could improve the overall result
+
+- Take care of the minutes of the meetings, including recording of all decisions. The minutes and presentations given in WG should be available in the WG Drive
+
+- Participate and on the preparation of technical drafts or documentation.
```
Comment (sffc) [inline guidelines/chair-group.md]: Just say, "Prepare technical drafts and documentation"

File: guidelines/chair-group.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+- Ensure that the composition of the WG is well balanced in order that a high number of representatives are involved in the work of WG.
+
+- Encourage the participants, attending the meeting and to collaborate actively.
+
+- Manage and accept WG applications.
+
+- Manage and organize all WG communication channels, email, mail group, boards and Github repositories
+  
+- Prioritize, label and organize the tasks of the WG.
+
```
Comment (sffc) [inline guidelines/chair-group.md]: New bullet suggestion:

"Formulate concrete questions that the WG can answer in their monthly meetings."

File: guidelines/chair-group.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
+## Chair group
+
+#### Definition and Motivation
+
+The Chair Group is a representative body of the Message Format WG that consists in spreading of the WG responsibilities among WG members that volunteered to get involved in the group chairing activities. Being part of a chair group requires an additional commitment that comes with a set of duties/responsibilities.
```
Comment (echeran) [inline guidelines/chair-group.md]: Replace "consists in spreading of" with "collectively shares" ?

File: guidelines/chair-group.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+
+#### Responsibilities
+
+- Ensure that the composition of the WG is well balanced in order that a high number of representatives are involved in the work of WG.
+
+- Encourage the participants, attending the meeting and to collaborate actively.
```
Comment (echeran) [inline guidelines/chair-group.md]: Remove "and" here.

File: guidelines/chair-group.md (hunk 2 above)
Comment (echeran) [inline guidelines/chair-group.md]: Change "and on" to "in".

File: guidelines/chair-group.md [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+- Participate in activities to share and promote the WG and all possible synergies that could improve the overall result
+
+- Take care of the minutes of the meetings, including recording of all decisions. The minutes and presentations given in WG should be available in the WG Drive
+
+- Participate and on the preparation of technical drafts or documentation.
+
+
+#### Participate
+
+Share your disposal to participate using one of the following using one of the following channels:
```
Comment (echeran) [inline guidelines/chair-group.md]: Change "disposal" to "interest".  Also, +1 to Shane's comment on having a different email address for that (not the main list).

File: guidelines/chair-group.md (hunk 1 above)
Comment (romulocintra) [reply guidelines/chair-group.md]: Something like :
```suggestion
- Send an email [email](mailto:message-format-cg@chromium.org)
```
?

File: guidelines/chair-group.md (hunk 2 above)
Comment (romulocintra) [reply guidelines/chair-group.md]:
```suggestion
- Prepare technical drafts and documentation.
```

File: guidelines/chair-group.md (hunk 1 above)
Comment (sffc) [reply guidelines/chair-group.md]: I think it's too easy to get `wg` and `cg` mixed up.  I made an alias `message-format-chairs@chromium.org`

File: guidelines/chair-group.md [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+- Prepare technical drafts and documentation.
+  
+- Formulate concrete questions that the WG can answer in their monthly meetings.
+
+#### Participate
+
+Share your interest to participate using one following channels:
+
+- Send an email [email](mailto:message-format-chairs@chromium.org)
+- Contact the [Mail Group](https://groups.google.com/a/chromium.org/forum/#!forum/message-format-wg)
```
Comment (romulocintra) [inline guidelines/chair-group.md]: Should we replace :
```suggestion
- Contact the [Mail Group](https://groups.google.com/a/chromium.org/forum/#!forum/message-format-chairs) ? 
```

Comment (romulocintra) [reply guidelines/chair-group.md]: cc @sffc

File: guidelines/chair-group.md [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+- Prepare technical drafts and documentation.
+  
+- Formulate concrete questions that the WG can answer in their monthly meetings.
+
+#### Participate
+
+Share your interest to participate using one following channels:
+
+- Send an email [email](mailto:message-format-chairs@chromium.org)
+- Contact the [Mail Group](https://groups.google.com/a/chromium.org/forum/#!forum/message-format-chairs) ?
```
Comment (romulocintra) [inline guidelines/chair-group.md]:
```suggestion
- Contact the [Mail Group](https://groups.google.com/a/chromium.org/forum/#!forum/message-format-chairs)
```

File: guidelines/chair-group.md [hunk 9]
```diff
@@ -0,0 +1,36 @@
...
+
+- Take care of the minutes of the meetings, including recording of all decisions. The minutes and presentations given in WG should be available in the WG Drive.
+
+- Prepare technical drafts and documentation.
+  
+- Formulate concrete questions that the WG can answer in their monthly meetings.
+
+#### Participate
+
+Share your interest to participate using one following channels:
```
Comment (romulocintra) [inline guidelines/chair-group.md]:
```suggestion
Share your interest to participate using one of the following channels:
```

**Pull request merged**